pkg/repository/manifest.go

```diff
@@ -91,12 +164,36 @@ type Snapshot struct {
 	Meta map[string]FileVersion `json:"meta"`
 }
 
+// NewSnapshot creates a Snapshot object.
+func NewSnapshot(initTime time.Time) *Snapshot {
+	return &Snapshot{
+		SignedBase: SignedBase{
+			Ty:          "snapshot",
```

Comment:
Can we define some constants for those literals?

Reply:
Yes, my fault - I've been meaning to do that